Tools/CISupport/ews-build/is_failing.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
```

Comment:
Would be good to keep the script name generic, like `results_db.py` or something like that. Currently it handles the case of pre-existing failures, but we would soon want to add the case for determining if a test is flaky failure (as ews encounters a lot of flaky tests). Similarly, there might be more functionality needed for results db interactions.

Reply:
This actually does the flaky case already. The endpoint we're hitting gives weights based on how far from the specified commit results are coming from.